modules/fakemeta/engfunc.cpp

```diff
@@ -595,8 +596,8 @@ static cell AMX_NATIVE_CALL engfunc(AMX *amx, cell *params)
 	case	EngFunc_LightStyle:			// void )			(int style, const char* val);
 		cRet = MF_GetAmxAddr(amx,params[2]);
 		iparam1=cRet[0];
-		temp = MF_GetAmxString(amx,params[3],0,&len);
-		(*g_engfuncs.pfnLightStyle)(iparam1,temp);
+		string = MF_GetAmxString(amx,params[3],0,&len);
+		(*g_engfuncs.pfnLightStyle)(iparam1, string.chars());
```

Comment:
`string` is temporary, `LightStyle` requires static string.

Reply:
What about `ALLOC_STRING`? Or should we use a global var specifically for that?